app/assets/stylesheets/components/_organizations.scss

```diff
@@ -0,0 +1,70 @@
+.organization-details {
+  h2,
+  h3,
+  h4,
+  h5 {
+    font: inherit;
+    margin: 0;
+  }
+
+  h2,
+  h4,
+  h5 {
+    font-weight: 700;
+  }
+
+  h2 {
+    font-size: 20pt;
+  }
+
+  body {
+    line-height: 1;
+  }
+
+  .border-above {
+    border-top: 1px solid #bbb;
+    margin-bottom: 15pt;
+    padding-top: 8pt;
+  }
+
+  .border-around {
+    border: 1px solid #bbb;
```

Comment:
Color literals like `#bbb` should only be used in variable declarations; they should be referred to via variable everywhere else.